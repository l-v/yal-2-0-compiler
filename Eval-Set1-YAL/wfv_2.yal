
module wfv_2
{

 function s = zeros(a[])
 {
   i = 0;
   while(i < a.size) {
      a[i] = 0;
      i = i + 1;
   }

   s = a.size;
 }


 function main()
 {

   a = [3];
   size_a = zeros(a);

   io.println("numero de elementos de a: ",size_a); // result= 3

 }
}
