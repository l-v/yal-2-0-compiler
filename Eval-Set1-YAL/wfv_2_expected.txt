numero de elementos de a: 3
